src/test/kotlin/uk/gov/justice/digital/hmpps/courtregister/resource/CourtResourceTest.kt

```diff
@@ -190,7 +189,17 @@ class CourtResourceTest : IntegrationTest() {
             user = "bobby.beans"
           )
         )
-        .body(BodyInserters.fromValue(CourtDto("ACCRYD", "A New Court", "a description", CourtTypeDto("YOUTH", "Youth Court"), true)))
```

Comment:
This should be an InsertCourtDto, but I've gone with a map to make sure the deserializing works.